README.md

```diff
@@ -60,26 +91,39 @@ This plugin has been inspired by the work previously done by [esensar](https://g
     remove_existing_container = true,
     -- By default, if no extra config is added, following nvim_dotfiles are
     -- installed: "https://github.com/LazyVim/starter"
-    -- This is an example for configuring other nvim_dotfiles inside the docker container
-    nvim_dotfiles_repo = "https://github.com/arnaupv/dotfiles.git",
-    nvim_dotfiles_branch = "main", -- branch to clone from nvim_dotfiles_repo 
-    nvim_dotfiles_install_command = "cd ~/nvim_dotfiles/ && ./install.sh",
-    -- In case you want to change the way the devenvironment is setup, you can also provide your own setup
-    setup_environment_repo = "https://github.com/arnaupv/setup-environment",
-    setup_environment_install_command = "./install.sh -p 'nvim stow zsh'",
+    -- This is an example for configuring other dotfiles inside the docker container
+    dotfiles_repository = "https://github.com/erichlf/dotfiles.git",
+    dotfiles_branch = "main", -- branch to clone from dotfiles_repository`
+    dotfiles_targetPath = "~/dotfiles", -- location to install dotfiles
+    dotfiles_targetPath = "install.sh", -- script to run after dotfiles are cloned
   },
   keys = {
     -- stylua: ignore
     {
-      "<leader>cdu",
+      "<leader>Du",
       ":DevcontainerUp<cr>",
-      desc = "Up the DevContainer",
+      desc = "Bring up the DevContainer",
     },
     {
-      "<leader>cdc",
+      "<leader>Dc",
       ":DevcontainerConnect<cr>",
       desc = "Connect to DevContainer",
     },
+    {
+      "<leader>De",
```

Comment:
`<Leader>De` results in:

	Error executing Lua callback: vim/shared.lua:0: Expected table, got string.
	stack traceback:
		[C]: in fucntion 'assert'
		vim/shared.lua: in function 'tbl_isempty'
		/ust/share/nvim/runtime/lua/vim/ui.lua:92: in function 'input'
		...ontainer-cli/lua/devcontainer_cli/devcontainer_utils.lua:179: in fuction `exec`
		...vcontainer-cli/lua/devcontainer_cli/devcontainer_cli.lua:38 in function <...vcontainer-cli/lua/devcontainer_clie/devcontainer_cli.lua:25>

Weirdly `<M-4>` does bringup a terminal connected to the container. I checked to see if I removed previous keymaps, I think I did, but not 100% sure

Reply:
I am unable to reproduce this, so any extra info would be helpful.

Reply:
This seems to be fixed now.